runtime/base/base.Dockerfile

```diff
@@ -254,8 +254,6 @@ RUN set -xe; \
     --enable-http \
     --enable-ftp  \
     --enable-file \
-    --enable-ldap \
```

Comment:
This is broken in master too, just failing silently instead of loudly. Adding ldap support out of scope of this PR.

```
2021-11-11T18:09:10.1163156Z checking for LDAP libraries... [91mconfigure: WARNING: Cannot find libraries for LDAP support: LDAP disabled`
```